frontends/web/src/api/bitbox02.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { apiGet, apiPost } from '../utils/request';
+import { SuccessResponse, FailResponse } from './response';
+
+export type DeviceInfo = {
+    initialized: boolean;
+    mnemonicPassphraseEnabled: boolean;
+    name: string;
+    securechipModel: string;
+    version: string;
+}
+
+type DeviceInfoResponse = {
+    deviceInfo: DeviceInfo;
+} & SuccessResponse;
```

Comment:
forgot this comment. not a big deal but pesonally i'd prefer for `SuccessResponse` to be in front because it's just easier to spot, like this:

```ts
type DeviceInfoResponse = SuccessResponse & {
    deviceInfo: DeviceInfo;
};
```

but @thisconnect up to you.

Reply:
💯  looks better, squashed into https://github.com/digitalbitbox/bitbox-wallet-app/compare/9eb6607f64abb0106ce8e154ac59faab236efab4..c0544ed4d999040634d9482e09d61bbf65c74037